src/rules/value-no-redundant-longhand/README.md

```diff
@@ -0,0 +1,58 @@
+# value-no-redundant-longhand
+
+Disallow redundant values in longhand properties.
+
+```css
+.foo { margin: 1px 1px 1px 1px; }
+/**            ↑
+ *  These properties */
+```
+
+This rule warns you when you use redundant values in the following longhand properties:
```

Comment:
"This rule warns you when you use redundant longhand values in the following  properties"